packages/internal/tests/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+	"extends": "../../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": ".",
+		"noEmit": true,
+		"types": ["vitest/globals"]
+	},
+	"include": [".", "../src"],
```

```suggestion
	"include": ["."],
```

This _should_ not be necessary